### ISSUE TITLE
A11y_Visual Studio Code Client_Search_Global Search_Keyboard: We are not able to access the tooltips present for "Match case", "Match whole word" etc present in "global search" section via keyboard.

### ISSUE THREAD

**Issue by Rama8345** [microsoft/vscode#203320]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG2.1.1; #Keyboard;

### Environment and OS details:
Visual Studio Code insider version 1.86.0
Microsoft Windows 11 Enterprise 22H2 Build 22621.3007

### Repro Steps:

1. Open Visual Studio Code Insiders editor
2. Using keyboard navigation View Menu Bar
3. TAB till search option and press ENTER.
4. TAB till "Match case", "Match whole word" etc controls.
5. Observed that We are not able to access the tooltips present for "Match case", "Match whole word" etc present in "global search" section via keyboard.

### Actual Result:
We are not able to access the tooltips present for "Match case", "Match whole word" etc present in "global search" section via keyboard.

**Similar issue is also observed throughout the application.**

### Expected Result:
We should be able to access the tooltips present for "Match case", "Match whole word" etc present in "global search" section via keyboard.

### User Impact:
Keyboard only users will not be able to get tooltip information of the control.

### Attachment:
![keyboard](https://github.com/microsoft/vscode/assets/115216656/e4a1867b-c942-458b-aaae-294ad5c29911)

![Bug_203320_Note](https://github.com/microsoft/vscode/assets/115216656/792f350a-df4c-4ee3-afdf-44404ee1c3a7)
